go/ql/lib/semmle/go/Decls.qll

```diff
@@ -416,8 +416,19 @@ class AliasSpec extends @aliasspec, TypeSpec { }
 class TypeDefSpec extends @typedefspec, TypeSpec { }
 
 /**
- * A field declaration, of a struct, a function (in which case this is a parameter or result variable),
- * or an interface (in which case this is a method or embedding spec).
+ * A field declaration, in a struct, a function (for parameter or result
+ * variables), or an interface (in which case this is a method or embedding
+ * spec).
+ *
+ * Examples:
```

Comment:
These examples and those below are just context-free variable declarations -- I'd suggest showing them in context using `[]` or such to highlight what you're talking about: for example, `func f([x, y int]) [int]`. This would mainly be useful for `Field*` to highlight that this could be field-of-struct or parameter-of-method or ...